notebook.ipynb

```diff
@@ -2,7 +2,15 @@
  "cells": [
```

Comment:
<p>More a note than a suggestion. Pytorch has an option to run in deterministic mode (see <a href="https://pytorch.org/docs/2.3/generated/torch.use_deterministic_algorithms.html" target="_blank">https://pytorch.org/docs/2.3/generated/torch.use_deterministic_algorithms.html</a>), is there something similar for Tensorflow?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/67a1e320-7c47-4ea9-8df8-e868326bc90b/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Reply:
<p>This is something that was looked into, but some of the underlying methods required within Tensorflow did not support it. And, as with pytorch's approach, it would be deterministic only for the same hardware and come at a performance deficiency.</p><p> </p><p>More details here:</p><p><a href="https://github.com/icenet-ai/icenet/issues/8" target="_blank">https://github.com/icenet-ai/icenet/issues/8</a></p>